tests/unit/components/playback/controls/PlateBarcode.spec.js

```diff
@@ -57,176 +57,37 @@ describe("PlateBarcode.vue", () => {
-    wrapper.find("input").setValue("MA200440012");
-    await wrapper.vm.$nextTick(); // wait for update
-    // confirm pre-condition
-    expect(store.state.playback.barcode).not.toBeNull();
-    store.commit(
-      "playback/set_playback_state",
-      playback_module.ENUMS.PLAYBACK_STATES.CALIBRATING
+    expect(wrapper.find("input").element.value).toEqual("MA200440004");
+    expect(wrapper.find(".input__plate-barcode-entry-valid").isVisible()).toBe(
+      true
     );
-    await wrapper.vm.$nextTick(); // wait for update
-    expect(wrapper.vm.platebarcode).toEqual("MA200440012");
   });
 
-  test("Given that the User entered a valid barcode of 11 characters, When user tries to enter barcode with an additional 12th digit, then it is not considered valid in Vuex", async () => {
-    wrapper.find("input").setValue("MA200440012"); // test case will fail on delating if (barcode_len >= 10 && barcode_len < 12) in API validatePlateBarcode()
-    await wrapper.vm.$nextTick(); // wait for update
-    // confirm pre-condition
-    expect(store.state.playback.barcode).not.toBeNull();
-    wrapper.find("input").setValue("MA2004400122");
-    await wrapper.vm.$nextTick(); // wait for update
-    expect(store.state.playback.barcode).toBeNull();
-  });
-  test("Set a proper platecode [input.length = 11] has and validate that no the red squiggle line is not present", async () => {
-    wrapper.find("input").setValue("M120044099");
-    await wrapper.vm.$nextTick(); // wait for update
-    expect(wrapper.find("input").html()).toContain('spellcheck="false"');
-  });
-  test("Fire an event to paste text ABCD validate its not updated on input", async () => {
-    const mEvent = {
-      clipboardData: { getData: jest.fn().mockReturnValueOnce("12") },
-    };
-    wrapper.find("input").trigger("paste", mEvent);
-    await wrapper.vm.$nextTick(); // wait for update
-    expect(wrapper.find("input").value).toBeUndefined();
-    expect(wrapper.find("input").html()).toContain('onpaste="return false;"');
-  });
-  test("On Vuex Store for playback_state set to RECORDING input is set to readonly true", async () => {
-    store.commit(
-      "playback/set_playback_state",
-      playback_module.ENUMS.PLAYBACK_STATES.CALIBRATED
-    );
-    await wrapper.vm.$nextTick(); // wait for update
-    // confirm pre-condition
-    expect(wrapper.find("input").html()).not.toContain("disabled");
-    store.commit(
-      "playback/set_playback_state",
-      playback_module.ENUMS.PLAYBACK_STATES.RECORDING
-    );
-    await wrapper.vm.$nextTick(); // wait for update
-    expect(wrapper.find("input").html()).toContain("disabled");
-    wrapper.find("input").setValue("MA20044001");
+  test("Given a invalid barcode has been into the Vuex, When Playback State is at any state, Then the text of the Barcode Input field should be valid barcode string and Green Box is visible", async () => {
```

Comment:
same comment about the "When" phrasing as previous test

Reply:
Removed "When Playback State" its now "When system_status is in any state"